events.html

```diff
@@ -0,0 +1,212 @@
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="box box-yellow">
+            <div class="event">
+                <img class="event--img" src="https://mymodernmet.com/wp/wp-content/uploads/2019/03/elements-of-art-thumbnail-1.jpg">
+                <p class="event--title">Battle of The Canvas - Oakland</p>
+                <p class="event--text">Appearing for the first time in Oakland, the Battle Of The Canvas® originally produced in Philadelphia, continues to be a means to expose, support and celebrate local artistry in cities throughout the country. Simone Banks, co-founder of WAXXED and Andre Jones  have partnered to bring the live painting festival to Oakland, working hand in hand to provide access and visibility to some of the most talented artists the city has to offer. All artists will compete for a $500 Grand Prize, $350 in Gift Cards donated by Flax Art Store and city wide exposure.</p>
+                <div class="event--bottom">
+                    <p class="event--dateTitle">Date:</p>
+                    <p class="event--date">August 15th, 4 PM</p>
+                    <div class="event--btn">
+                        <a class="btnLink">Learn More</a>
+                    </div>
+                    <div class="event--btn2">
+                        <a class="btnLink btnLink-small">Learn More</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="box box-red">
+            <div class="event">
+                <img class="event--img" src="https://mymodernmet.com/wp/wp-content/uploads/2019/03/elements-of-art-thumbnail-1.jpg">
+                <p class="event--title">Battle of The Canvas - Oakland</p>
+                <p class="event--text">Appearing for the first time in Oakland, the Battle Of The Canvas® originally produced in Philadelphia, continues to be a means to expose, support and celebrate local artistry in cities throughout the country. Simone Banks, co-founder of WAXXED and Andre Jones  have partnered to bring the live painting festival to Oakland, working hand in hand to provide access and visibility to some of the most talented artists the city has to offer. All artists will compete for a $500 Grand Prize, $350 in Gift Cards donated by Flax Art Store and city wide exposure.</p>
+                <div class="event--bottom">
+                    <p class="event--dateTitle">Date:</p>
+                    <p class="event--date">August 15th, 4 PM</p>
+                    <div class="event--btn">
+                        <a class="btnLink">Learn More</a>
+                    </div>
+                    <div class="event--btn2">
+                        <a class="btnLink btnLink-small">Learn More</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <p class="sectionHeading l-center l-section">Past events...</p>
```

Comment:
- use `<h2 class="sectionHeading sectionHeading-spaced .sectionHeading-centered">` instead 
- and add  the tag `<section class="l-section">` around line 131-161